tomls/omnibus-arbitrum-mainnet/collaterals/susde.toml

```diff
@@ -0,0 +1,31 @@
+[var.susde_collateral_config]
+susde_max_collateral_limit = "<%= parseEther(String(3_000_000)) %>"
+susde_issuance_ratio = "<%= parseEther('2.00') %>"
```

Comment:
can you please increase this to 1000% https://sips.synthetix.io/sccp/sccp-352/

Reply:
done